src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-style: heavy;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  border: none;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big_search {
+  box-sizing: border-box;
+  background: #fff;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  font-family: inherit;
```

Comment:
no need for inherit font-family, it's inherited from the parent elements by default